development.cfg

```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    test-plone-4.3.x.cfg
+    test-plone-5.1.x.cfg
```

Comment:
I would also update the Trove classifiers in `setup.py` to declare Plone 5 compatibility

Reply:
Ah, yes, sure 👍